README.md

```diff
@@ -126,7 +129,7 @@ npm run start
 | POSTAGE_AMOUNT          | undefined                   | Postage stamp amount to be used when buying new stamps or selecting existing stamps.                       |
 | POSTAGE_USAGE_THRESHOLD | 0.7                         | Usage percentage at which new postage stamp will be bought (value between 0 and 1).                        |
 | POSTAGE_USAGE_MAX       | 0.9                         | Usage percentage at which existing postage stamp should not be considered viable ( values 0 to 1).         |
-| POSTAGE_TTL_MIN         | 5 \* POSTAGE_REFRESH_PERIOD | Minimal time to live for the postage stamps to still be considered for upload (in seconds).                |
+| POSTAGE_TTL_MIN         | 5 \* POSTAGE_REFRESH_PERIOD | In `autobuy`, Minimal time to live for the postage stamps to still be considered for upload (in seconds). In `extends TTL` is mandatory and above 60 seconds  |
```

Comment:
```suggestion
| POSTAGE_TTL_MIN         | 5 \* POSTAGE_REFRESH_PERIOD | In `autobuy`, minimal time to live for the postage stamps to still be considered for upload (in seconds). In `extends TTL` is mandatory and required to be above 60 seconds  |
```

Reply:
Also, the default value is bit misleading as it is not true for the `extendsTTL` mode, but not sure really how to represent that here 😅